content/en/docs/concepts/scheduling/scheduling-framework.md

```diff
@@ -146,8 +143,11 @@ state, it will either trigger [Unreserve](#unreserve) plugins (on failure) or
 
 ### Permit
 
-These plugins are used to prevent or delay the binding of a Pod. A permit plugin
-can do one of three things.
+The Permit extension point contains two parts which spans scheduling cycle and
+binding cycle.
+
+In the scheduling cycle, Permit plugins are called to prevent or delay the binding
+of a Pod. A permit plugin can do one of three things:
```

Comment:
```suggestion
Permit plugins are invoked at the end of the scheduling cycle of each pod to prevent or delay the binding to the candidate node. A permit plugin can do one of three things:
```